cmd/upgrade.go

```diff
@@ -168,6 +170,17 @@ func (uc *upgradeCmd) loadCluster() error {
 		return errors.Wrap(err, "error parsing the api model")
 	}
 
+	// Use the Windows VHD associated with the aks-engine version if upgradeWindowsVHD is set to "true"
+	if uc.upgradeWindowsVHD {
+		if uc.containerService.Properties.WindowsProfile.WindowsPublisher == api.AKSWindowsServer2019OSImageConfig.ImagePublisher && uc.containerService.Properties.WindowsProfile.WindowsOffer == api.AKSWindowsServer2019OSImageConfig.ImageOffer {
```

Comment:
this maybe more readable if `uc.containerService.Properties.WindowsProfile` is stored in a variable.

Reply:
updated